packages/babel-preset-gatsby/src/dependencies.ts

```diff
@@ -1,10 +1,11 @@
 // This file is heavily based on create-react-app's implementation
 // @see https://github.com/facebook/create-react-app/blob/master/packages/babel-preset-react-app/dependencies.js
 
-const path = require(`path`)
-const resolve = m => require.resolve(m)
+import path from "path"
 
-module.exports = function(api, options = {}) {
+const resolve = (m: string): string => require.resolve(m)
+
+export default (): object => {
```

Comment:
object is definitely not accurate here. But I think it's fine since this is just consumed by babel

Reply:
Yeah I couldn't find any typings in a `@types/x` package.
If you know where I could find it, I'm happy to use it 🙂